grpc/go-cubic/go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/cobaltspeech/sdk-cubic/grpc/go-cubic
 go 1.12
```

Comment:
Should we bump this number?

Reply:
i don't think it's a problem right now (it's the minimum version needed i think)